cadence/templates/server-configmap.yaml

```diff
@@ -162,14 +162,16 @@ data:
       {{- $currentClusterName := .Values.server.config.clusterMetadata.currentClusterName }}
       {{- $currentClusterIndex := 0 }}
       {{- $frontendComponentName := (include "cadence.componentname" (list . "frontend")) }}
-      {{- $serverFrontendServicePort := .Values.server.frontend.service.port }}
+      {{- $serverFrontendServiceGRPCPort := .Values.server.frontend.service.grpcPort }}
+      {{- $serverFrontendServiceTransport := .Values.server.frontend.service.transport }}
       {{- range $clusterIndex, $clusterInfo := .Values.server.config.clusterMetadata.clusterInformation }}
         {{- if eq $clusterInfo.name $currentClusterName }}{{ $currentClusterIndex = $clusterIndex }}{{ end }}
         {{ $clusterInfo.name }}:
           enabled: {{ $clusterInfo.enabled }}
           initialFailoverVersion: {{ $clusterIndex }}
           rpcName: "{{ $frontendComponentName }}"
-          rpcAddress: "{{ $clusterInfo.rpcAddress | default (printf "%s%s%.0f" $frontendComponentName ":" $serverFrontendServicePort ) }}"
+          rpcAddress: "{{ $clusterInfo.rpcAddress | default (printf "%s%s%.0f" $frontendComponentName ":" $serverFrontendServiceGRPCPort ) }}"
+          rpcTransport: "{{ $serverFrontendServiceTransport }}"
```

Comment:
**Question**: Is this going to work even for other transports than GRPC? I mean using the GRPCPort.

Reply:
Thanks for the question! I reworked this part, now the default is `grpc` transport with port `7833` but the transport is `tchannel` (the old default) the default port is still going to be `7933`.